rita_bin/src/client.rs

```diff
@@ -53,6 +53,8 @@ fn main() {
     .unwrap_or_else(|e| e.exit());
 
     let settings_file = args.flag_config;
+    println!("Settings file {}", settings_file);
```

Comment:
I used a println because this generally happens early in main() before logging is setup.  I think it's useful enough info to always have once at the beginning of an app.  The message could be improved though.